adal/src/main/java/com/microsoft/aad/adal/StringExtensions.java

```diff
@@ -46,19 +46,22 @@
 
     private static final String TOKEN_HASH_ALGORITHM = "SHA256";
 
+    private StringExtensions() {
+        // Intentionally left blank
+    }
     /**
      * checks if string is null or empty.
      * 
      * @param param String to check for null or blank
      * @return boolean if the string was null or blank
      */
-    static boolean IsNullOrBlank(String param) {
-        return (param == null || param.trim().length() == 0);
+    static boolean isNullOrBlank(String param) {
+        return param == null || param.trim().length() == 0; //NOPMD
```

Comment:
you mean explain why we are not looping through the string to check for all spaces?

---
In reply to: [73039245](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/689#discussion_r73039245) [](ancestors = 73039245)